app/src/main/java/org/p2p/wallet/user/interactor/UserInteractor.kt

```diff
@@ -122,6 +122,20 @@ class UserInteractor(
         }
     }
 
+    suspend fun loadUserRatesIfEmpty(userTokens: List<Token.Active>) {
+        when {
+            // if prices are not loaded at all, load them
+            !userLocalRepository.arePricesLoaded() -> {
+                loadUserRates(userTokens)
```

Comment:
do you want to try catch that? Error while loading user rates should not block other request and startPolling()